<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DELL-C0HKRC2" failures="5" tests="6" name="Test" time="28.673" errors="0" timestamp="2022-03-21T20:48:39 ART">
  <testcase classname="pruebas.DataProviderTest" name="login" time="8.96"/>
  <testcase classname="pruebas.DataProviderTest" name="login" time="3.586">
    <failure type="java.lang.AssertionError" message="expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]">
      <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]
at pruebas.DataProviderTest.login(DataProviderTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="pruebas.DataProviderTest" name="login" time="3.676">
    <failure type="java.lang.AssertionError" message="expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]">
      <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]
at pruebas.DataProviderTest.login(DataProviderTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="pruebas.DataProviderTest" name="login" time="3.841">
    <failure type="java.lang.AssertionError" message="expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]">
      <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]
at pruebas.DataProviderTest.login(DataProviderTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="pruebas.DataProviderTest" name="login" time="3.72">
    <failure type="java.lang.AssertionError" message="expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]">
      <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]
at pruebas.DataProviderTest.login(DataProviderTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="pruebas.DataProviderTest" name="login" time="3.47">
    <failure type="java.lang.AssertionError" message="expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]">
      <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]
at pruebas.DataProviderTest.login(DataProviderTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Test -->
